lib/src/modern_player_controls.dart

```diff
@@ -136,8 +138,60 @@ class _ModernPlayerControlsState extends State<ModernPlayerControls> {
 
   /// Get audio and subtitle tracks
   void _getTracks() async {
-    _audioTracks = await player.getAudioTracks();
-    _subtitleTracks = await player.getSpuTracks();
+    // Run in parallel - they don't depend on each other.
+    final tracksFutures = Future.wait([
+      player.getAudioTracks(),
+      player.getSpuTracks(),
+    ]);
```

Comment:
I modified these to run in parallel.

Reply:
Thank you for this, I didn't have any idea about this before. :)